gnucash/import-export/ofx/gnc-ofx-import.c

```diff
@@ -1042,9 +1043,15 @@ void gnc_file_ofx_import (GtkWindow *parent)
     DEBUG("gnc_file_ofx_import(): Begin...\n");
 
     default_dir = gnc_get_default_directory(GNC_PREFS_GROUP);
+    GList *filters = NULL;
+    GtkFileFilter* filter = gtk_file_filter_new ();
```

Comment:
This is a C file and declaring variables after code isn't legal in C. Move these lines to the declarations block above.